invenio_app_rdm/config.py

```diff
@@ -36,6 +37,10 @@
 from flask_principal import Denial
 from invenio_access.permissions import any_user
 from invenio_rdm_records.config import RDM_NAMESPACES
+from invenio_stats.aggregations import StatAggregator
+from invenio_stats.contrib.event_builders import build_file_unique_id
+from invenio_stats.processors import EventsIndexer, anonymize_user, flag_robots
+from invenio_stats.queries import ESTermsQuery
```

Comment:
nit: what does `ES` prefix stand for in `ESTermsQuery` ?

Reply:
That would be short for "**ES** is the only search engine we ever imagine supporting"...

/s

I guess this could be dropped to be consistent with our generalizations for search backends.

Reply:
in this case I think now it is the time, since we are transitioning, @max-moser is it possible to change the naming at this point ?

Reply:
oops, yeah, missed that one while i was at it :sweat_smile: 
sure, i'd suggest renaming it to `SearchTermsQuery` etc. and maybe providing back-compat aliases similar to [this guy here](https://github.com/inveniosoftware/invenio-stats/blob/master/invenio_stats/receivers.py#L75-L76)?

Reply:
on second thought, let's ~~not go to camelot – it's a silly place~~ just strip the `ES` prefix?
here's the [PR](https://github.com/inveniosoftware/invenio-stats/pull/131) for the renaming, with a bunch of other elasticsearch references being fixed too